inference-engine/src/readers/ir_reader/ie_ir_parser.cpp

```diff
@@ -692,21 +677,6 @@ std::shared_ptr<ngraph::Node> V10Parser::XmlDeserializer::createNode(
     // Find registerd opset
     auto opsetIt = opsets.find(params.version);
 
-    if (isDefaultOpSet(params.version)) {
```

Comment:
@ilyachur I think it is definitely needs your review.

Reply:
It is ok, because we removed all creators.